### PR TITLE
APPS-2428 - Adding AAR publishing support

### PR DESCRIPTION
**Pull request by smpete**

### CODE REVIEW

File: layer-atlas/build.gradle [hunk 1]
```diff
@@ -49,4 +62,76 @@ dependencies {
...
+            name = pomProjectName
+            packaging "aar"
+            artifactId = artifactIdName
+
+            developers {
+                developer {
+                    name 'Amar Srinivasan'
+                    email 'amar@layer.com'
+                }
+                developer {
```
Comment (sriamar) [inline layer-atlas/build.gradle]: Remove Doug from here. He is not involved in Atlas (or Android) anymore.

**Comment by smpete** [comment]
Let's hold up on this for the moment. The `publishReleaseNotes` task is getting executed on a gradle sync. Looking into it now.

**Comment by smpete** [comment]
Ok should be good to be looked at again.

File: README.md [hunk 2]
```diff
@@ -68,7 +68,20 @@ The Atlas library is located in the `layer-atlas` directory.  The table below de
...
+    maven { url "https://raw.githubusercontent.com/layerhq/releases-android/master/releases/" }
+    maven { url "https://raw.githubusercontent.com/layerhq/Atlas-Android/master/releases/" }
+}
+
+dependencies {
+    compile 'com.layer.atlas:layer-atlas:0.2.10'
+}
+```
+
+Additional installation information can be found at https://github.com/layerhq/releases-android-atlas
```
Comment (sriamar) [inline README.md]: I guess this is changing to releases-android now?

File: layer-atlas/build.gradle [hunk 3]
```diff
@@ -23,9 +30,16 @@ android {
 
 repositories {
     maven { url "https://raw.githubusercontent.com/layerhq/releases-android/master/releases/" }
+//    maven { url "/Users/work/.gitRepos/layerhq/releases-android/releases" }
```
Comment (sriamar) [inline layer-atlas/build.gradle]: remove this?

**Comment by sriamar** [comment]
@smpete Left a comment about link. Take a look, and feel free to merge.

File: layer-atlas/build.gradle (hunk 3 above)
Comment (smpete) [inline layer-atlas/build.gradle]: Nice catch. In my excitation of getting this to work I apparently committed everything.

**Pull request merged**